File: content/examples/other/leetcode/problems/missing-number/example-1/solution.js
function missingNumberSort(nums) {
  nums.sort((a, b) => a - b);
  for (let i = 0; i < nums.length; i += 1) {
    if (nums[i] !== i) return i;
  }
  return nums.length;
}

if (require.main === module) {
  console.log(missingNumberSort([3, 0, 1]));
}

module.exports = { missingNumberSort };

File: content/examples/other/leetcode/problems/missing-number/example-1/Solution.java
import java.util.Arrays;

public class Solution {
  static int missingNumberSort(int[] nums) {
    Arrays.sort(nums);
    for (int i = 0; i < nums.length; i += 1) {
      if (nums[i] != i) return i;
    }
    return nums.length;
  }

  public static void main(String[] args) {
    System.out.println(missingNumberSort(new int[] {3, 0, 1}));
  }
}
